packages/vite/src/node/utils.ts

```diff
@@ -441,7 +441,7 @@ export function emptyDir(dir: string, skip?: string[]): void {
     const abs = path.resolve(dir, file)
     // baseline is Node 12 so can't use rmSync :(
     if (fs.lstatSync(abs).isDirectory()) {
-      emptyDir(abs)
+      emptyDir(abs, skip)
```

Comment:
The comment in the function says "Pass an optional `skip` array to preserve files in the root directory.". We should change it to "Pass an optional `skip` array to preserve certain files.". The GitHub UI don't allow suggestions in unmodified lines :/

Reply:
Oh, it's intentional? I just had a weird condition racing issue on Nuxt with Vitest. This is not the fix to my specific problem but I just read them and thought it was a mistake. Maybe let's keep it as-is then.